frontend/pages/hosts/details/DeviceUserPage/DeviceUserPage.tsx

```diff
@@ -260,6 +260,11 @@ const DeviceUserPage = ({
       refetchOnWindowFocus: false,
       retry: false,
       onSuccess: ({ host: responseHost }) => {
+        // If we're just showing the setup screen,
+        // we don't need to refetch or alert on offline hosts.
+        if (location.query.setup_only) {
+          return;
+        }
```

Comment:
This page doesn't use any host information that we'd be concerned about getting stale. It really just needs the license and the host platform.